nautobot/extras/filters/__init__.py

```diff
@@ -902,6 +906,53 @@ class Meta:
         ]
 
 
+class JobQueueFilterSet(BaseFilterSet, CustomFieldModelFilterSetMixin):
```

Comment:
Should this use `NautobotFilterSet` instead of `BaseFilterSet`?

Probably should use `TenancyModelFilterSetMixin` as well.